.gitattributes

```diff
@@ -7,4 +7,5 @@
 .travis.yml export-ignore
 tests/ export-ignore
 phpunit.xml export-ignore
+psalm.xml export-ignore
```

Comment:
I think `/.github` folder should be ignored during dist mode.

Reply:
I think it is ignored now though.